framework/utils/InputData.py

```diff
@@ -35,8 +35,72 @@ class Quantity:
   one = (1,1)
   one_to_infinity = (1,2)
 
-#
-#
+
+class CheckClass(object):
+  """
+  This checks to figure out if a node is a ParameterInput type
+  If the check passes, then it is.
+  """
+
+  def check(self, node):
+    """
+      Checks the node to see if it matches the checkDict
+      @ In, node, xml node to check
+      @ Out, bool, true if matches
+    """
+    assert False, "check function not implemented"
+
+
+  def failCheckReason(self, node):
+    """
+      returns a string about why the check failed
+      @ In, node, xml node to check
+      @ Out, string, message for user about why check failed.
```

Comment:
docstrings are not standard (same comments of above) .it is valid for all the others methods/docstrings

Reply:
Fixed.